### DISCUSSION TITLE
Add command-line argument parsing

### DISCUSSION THREAD

**Discussion by quantum9Innovation** [creamcropdev/creamcrop#81] [Ideas]
By default, implement certain automatic behaviors that can be turned off via command-line flags. For example, instead of waiting for the user to click the link shown in the terminal, auto open the localhost. Additionally, creamcrop should not log every single request and instead show only the required text in the terminal. There should be two flags `--no-open (n)` and `--verbose (v)` to change creamcrop's default behavior. The `n` flag will tell creamcrop to let the user open the page and the `v` flag will tell creamcrop that it's okay to bombard the terminal with unnecessary text.

**Comment by Quantalabs** [comment]
Finished implementing in #82. If --verbose output is not enabled, only 2 lines are shown:
```
$ cream serve ./
Found config file, generating website...
Server running at http://localhost:8080
```
